components/card/card.css

```diff
@@ -0,0 +1,215 @@
+.card{
+    display: flex;
+    flex-direction: column;
+    width: 300px;
+    height: max-content;
+    border-radius: 4px;
+    background: var(--white);
+    border: 1px solid var(--off-grey);
```

Comment:
instead of naming the variables white and off-grey u could also name them according to the theme of ur site say --bg-color